ref/general/hardening-intro.adoc

```diff
@@ -45,9 +45,9 @@ If you use a custom registry, use the `UserRegistry` class to implement the regi
 SSO provides better security and regulatory compliance.
 If you use SSO, maintain your SSO configuration in a separate include file, not in the `server.xml` file.
 For more information about configuring SSO, see link:/docs/ref/general/#sso-config-json.html[SSO configuration with JSON Web Tokens].
-* *Carefully configure file permissions*: When you configure file permissions, ensure that the users who want access to the configuration fall into the appropriate permission bit.
-Users that aren't owners and aren't part of the authorized group fall into the `other` permission bit, which is indicated with `0`.
-* *Encrypt passwords*: Use encryption for passwords instead of Base64 encoding.
+* *Carefully configure file permissions*: When you configure file permissions, ensure that users who require access to the server's configuration have no more than the required capability (ie: a database administrator only has access to the data source configuration that is referenced in the `server.xml` using an `include` statement).
+Users that aren't administrators or developers with specialized access (ie: to view application/server logs), should fall into the `other` group for server configuration files, and all files should have the other group configured with no access (indicated with `0`).
```

Comment:
I tweaked this section a bit to remove the sets of parentheses and abbreviations. Read through it and let me know if I didn't capture it as you wanted it.